src/main/java/com/prgrms/modi/common/oauth2/OAuth2AuthenticationSuccessHandler.java

```diff
@@ -64,7 +59,8 @@ public void onAuthenticationSuccess(
         }
     }
 
-    protected String determineTargetUrl(HttpServletRequest request, HttpServletResponse response, Authentication authentication) {
+    protected String determineTargetUrl(HttpServletRequest request, HttpServletResponse response,
+        Authentication authentication) {
```

Comment:
```suggestion
    protected String determineTargetUrl(
        HttpServletRequest request, 
        HttpServletResponse response,
        Authentication authentication
    ) {
```

길어서 내려간거 같은데 그냥 다 내려버리죠!